app/uk/gov/hmrc/claimvatenrolmentfrontend/services/JourneyService.scala

```diff
@@ -58,4 +58,13 @@ class JourneyService @Inject()(journeyConfigRepository: JourneyConfigRepository,
   def removePostcodeField(journeyId: String, authInternalId: String): Future[UpdateWriteResult] = {
     journeyDataRepository.removeJourneyDataFields(journeyId, authInternalId, Seq(PostcodeKey))
   }
+
+  def removeBox5Field(journeyId: String, authInternalId: String): Future[UpdateWriteResult] = {
+    journeyDataRepository.removeJourneyDataFields(journeyId, authInternalId, Seq(Box5FigureKey))
+  }
+
+  def removeLastMonthSubmittedField(journeyId: String, authInternalId: String): Future[UpdateWriteResult] = {
+    journeyDataRepository.removeJourneyDataFields(journeyId, authInternalId, Seq(LastMonthSubmittedKey))
```

Comment:
You don't need to make two calls into the database to remove these fields, you could pass in a sequence of datakeys that will remove both fields in one call

`Seq(Box5FigureKey, LastMonthSubmittedKey)`

Reply:
You could call the method removeAdditionalVatReturnFields?